### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by VaheStepanyan100**
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
-  Implement popups for both mobile and desktop screen sizes:

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @VaheStepanyan100, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have all the linters passed, kindly fix all the linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by VaheStepanyan100** [comment]
Hi @Olamarx,
Please help me with  ESlint error  (113:44  error  Missing trailing comma  comma-dangle)

**Review by AlphaNtihinduka — COMMENTED** [review]
Hi @VaheStepanyan100 ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
# Invalid Code Review Request ❌ 

- [ ] You have submitted a project where there are no commits for the colloborator. Kindly reach out to your peer and request for the contribution. You should also consider updating your readme with the contributor(authore) details. Since this is a pair programming activity, it will help you to gain collaboration skills.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html [hunk 1]
```diff
@@ -312,5 +204,30 @@ <h2>Contact me</h2>
...
+            <span>See live</span>
+            <img src="assets/seeLiveIcon.png" alt="See live" />
+          </button>
+          <button>
+            <span>See source</span>
+            <img width="16px" src="assets/Vector.png" alt="See source" />
+          </button>
+        </div>
+      </div>
+    </div>
```
Comment (Ibizugbe) [inline index.html:231]: Nice job so far👍 But just as [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) for this milestone, we are to generate the popup content and project section dynamically with javascript using the array of objects we have created to store data about our projects😉 Kindly take away this hardcoded popup and generate it dynamically in your `js` file👍
generate popup and projects section dynamically

File: detailsPopup.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+const skillInfoListItems = [
+  'CANOPY',
+  '<img src="assets/Counter.png" alt="bullet" />',
+  'Back End Dev',
+  '<img src="assets/Counter.png" alt="bullet" />',
+  '2015',
+];
+
+const languages = ['HTML', 'CSS', 'JavaScript'];
```
Comment (Ibizugbe) [inline detailsPopup.js:50]: Excellent job storing your project information in an array of objects just as [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), but let's ensure information about our project is stored inside the array of objects 😉 I can see you already have this stored in your array as `technologies` so there's no need to store it again. you can access the one you have created in the main object as this makes your code dry, neater, and shorter👍 Kindly ensure every information about your projects is accessed from the array of objects you have created for your projects so as to make your codebase maintainable and dry😉.

File: styles.css [hunk 3]
```diff
@@ -115,6 +116,16 @@ body {
...
+.detail-popup button {
+  background-color: white;
+  cursor: pointer;
+  border-radius: 8px;
+  margin-right: 10px;
+  border: 1px solid #6070ff;
+  width: 135px;
+  height: 48px;
+}
+
```
Comment (Ibizugbe) [inline styles.css:128]: Nice job implementing the functionalities of the popup👍 But your popup doesn't match the design🥲 Kindly take note of the comparison below and ensure your popup matches the design😉 add every element as seen in the design👍  This applies too for your mobile screen. do well to ensure your popup matches the design both on mobile and on desktop as it is required for this milestone👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|yours|design|

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @VaheStepanyan100 👋👋, 

tryagain


Good job so far!👍👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍.
- [x] github flow practice has been used👍.
- [x] Popup functionality (close and open) works properly👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (LionRouge1) [reply styles.css:128]: - [ ] As requested by the previeous reviewer, note the `background-color` of the space around the popup windows on Figma design, but it is not showing on your design. Kindly make sure your popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).

### Your design :point_down: 


### Figma's Design :point_down:

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @VaheStepanyan100, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost

## Suggested changes :recycle: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by divinecharlotte — APPROVED** [review]
# Hello @VaheStepanyan100     😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- good commit messages 👍 


## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**